content/blog/2018/07/2018-07-25-contributor-summit.adoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Join us at the Jenkins Contributor Summit"

```

Comment:
Maybe it makes sense to add location/date to the title

Reply:
Yep